README.md

```diff
@@ -102,6 +102,11 @@ In order to be considered "valid", an IPFS core implementation  must expose the
   - [Miscellaneous](/SPEC/MISCELLANEOUS.md)
   - [config](/SPEC/CONFIG.md)
 
+Language-specific interface definitions and tests are in their directories:
```

Comment:
You only find the tests on the `js/src` folder. The interface definitions remain at `/SPEC`